app/helpers/public_document_routes_helper.rb

```diff
@@ -82,23 +82,6 @@ def get_involved_url(options = {})
     Plek.website_root + get_involved_path(options)
   end
 
-  def take_part_page_path(object, options = {})
```

Comment:
Really nice, I know this won't show up on the code removal metrics but this kind of simplification is great